controller/controller.go

```diff
@@ -36,10 +41,10 @@ const (
 )
 
 type Config struct {
-	CRD       *apiextensionsv1beta1.CustomResourceDefinition
-	CRDClient *k8scrdclient.CRDClient
-	Informer  informer.Interface
-	Logger    micrologger.Logger
+	CRD            *apiextensionsv1beta1.CustomResourceDefinition
+	BackOffFactory func() backoff.Interface
```

Comment:
When we are breaking things can we drop `BackOffFactory`?

Reply:
Let me follow up on this in a separate PR.